src/helpers/rerun-pr-checks.ts

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright 2021 Expedia, Inc.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    https://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import * as core from '@actions/core';
+import { context } from '@actions/github';
+import { octokit } from '../octokit';
+
+export const rerunPrChecks = async () => {
+  /** grab owner in case of fork branch */
```

Comment:
Why do we need to do this here and not in any other helper?

Reply:
since there's no way to grab workflow runs on a PR at the moment, i'm filtering workflow runs by branch and grabbing only the first occurrence. if the `head` is on a fork, i need to query that repo instead. this sort of hinges on the assumption that the forked repo is publicly available